Fix grammar (#19078)
mdn/content: files/en-us/web/javascript/reference/strict_mode/index.md

fixed.newProp = 'ohai'; // throws a TypeError
```

> Third, strict mode makes attempts to throw Error when you delete undeletable properties (where before the attempt would have no effect):

Reviewer: ```suggestion
Third, attempts to delete undeletable properties throw in strict mode (where before the attempt would have no effect):
```
